docs/changelog.rst

```diff
@@ -43,7 +46,7 @@ Detectors
 * **Detectors now required to define an attribute called name**, which should be unique within a `Scrubber` instance.
 * **Detectors are now passed a locale argument to the Detector initialiser**.
 * `Detectors have an optional `supported_locale(locale)` function that returns a bool to indicate if a given `Detector` supports a `locale`.
```

Comment:
there's an additional open quote at the start of the line: Detectors have an optional...